string key			#Label of the value
string value		#Value to track
float32 errorlevel			#Errorlevel of the value